server/routes/api/choropleth.py

```diff
@@ -35,52 +35,70 @@
   url_prefix='/tools/choropleth'
 )
 
-@bp.route('/api')
-def choropleth_api():
+@bp.route('/values')
+def choropleth_values():
     """Returns data for geographic subregions for a certain statistical 
             variable.
 
-    API Params: #TODO(iancostello): Split API calls up or move to client.
-        statVar -> The statistical variable to download, as a string.
-        perCapita -> Whether to return the per-capita value, as a string of a 
-                boolean.
-        geoId -> The currently viewed geography to render, as a string.
-        level -> The subgeographic level to pull and display information for,
+    API Params:
+        geoId: The currently viewed geography to render, as a string.
+        level: The subgeographic level to pull and display information for,
                 as a string. Choices: Country, AdministrativeLevel[1/2], City.
-        mdom -> The measurement denominator to use if perCapita is true, as a 
+        statVar: The statistical variable to pull data about.
+
+    API Returns:
+        values: dictionary of geo to statistical variable values (as a float)
+            for all subgeos.
+    """
+    # Get required request parameters.
+    requested_geoId = request.args.get("geoId")
+    if not requested_geoId:
+        return jsonify({"error": "Must provide a 'geoId' field!"}, 400)
+    stat_var = request.args.get("statVar")
+    if not stat_var:
+        return jsonify({"error": "Must provide a 'statVar' field!"}, 400)
+    display_level = get_sublevel(requested_geoId, request.args.get("level"))
+
+    # Get all subgeos.
+    geos_contained_in_place = dc.get_places_in(
+            [requested_geoId], display_level).get(requested_geo, [])
+    values_by_geo = dc.get_stats(geos_contained_in_place, stat_var)
+
+    # Add to dictionary for response.
+    populations_by_geo = {}
+    for geo_id, payload in values_by_geo.items():
+        if "data" in payload:
+            value = next(iter(
+                        reversed(payload['data'].values())))
+            populations_by_geo[geo_id] = value
+
+    # Return as json payload.
+    return jsonify(populations_by_geo, 200)
+
+@bp.route('/geo')
+def choropleth_geo():
+    """Returns data for geographic subregions for a certain statistical 
+            variable.
+
+    API Params:
+        geoId: The currently viewed geography to render, as a string.
+        level: The subgeographic level to pull and display information for,
+                as a string. Choices: Country, AdministrativeLevel[1/2], City.
+        mdom: The measurement denominator to use if perCapita is true, as a 
```

Comment:
This should be stat var?

Reply:
Distinct from stat var, this is used for the per_capita option if you want to change it. For example you may divide the total number of unemployed by the total in labor force. The UI currently doesn't provide a way to customize this though.

Reply:
No, mdom refers to the measurement denominator used to divide if per_capita is true.

Reply:
E.g. one could select the mdom to be Count_Person_InLaborForce with a stat var of Count_Person_Employed